src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java

```diff
@@ -157,6 +163,11 @@ void store(WorkerResult wr) {
                     continue;
                 }
 
+                if (!metricTiers.isMetricEnabled(cd.checkId)) {
```

Comment:
Just an idea: rename the method to `isMetricDisabled`, to avoid negation during call.